### ISSUE TITLE
rbp2-headers package doesn't symlink in its postinst script

### ISSUE THREAD

**Issue by lowlyocean** [osmc/osmc#471]
See https://github.com/atar-axis/xpadneo/issues/50#issuecomment-435595673

It seems that users may encounter trouble with DKMS modules with OSMC since no link is made by the kernel headers package to "/lib/modules/`` `uname -r` ``/build"

**Comment by atar-axis** [comment]
> Maintainer: Sam G Nazarko <email@samnazarko.co.uk>

@samnazarko

**Comment by samnazarko** [comment]
We don't recommend DKMS because of the variation of kernels used.
I can fix the link; but don't recommend DKMS approach.

It might be worth us getting this module upstreamed. How does it compare to xpad module?

**Comment by atar-axis** [comment]
Yeah, would be nice if you could fix the *missing* link in the package :grin:

The driver is, at least as yet, complementary to xpad. There is no other driver around which does support the Xbox Wireless gamepad fully (it has a whole bunch of quirks).

**Comment by lowlyocean** [comment]
> (which has a whole bunch of quirks).
>
Quirks described https://github.com/paroj/xpad/issues/53#issuecomment-328280325

**Comment by lowlyocean** [comment]
Assuming there's some PPA with the packages but not sure if there's a platform to contribute. Marking this close on good faith

**Comment by samnazarko** [comment]
I will reopen it so it gets remembered
